nrpy/examples/hydro_without_hydro.py

```diff
@@ -0,0 +1,248 @@
+if (strncmp(commondata->outer_bc_type, "radiation", 50) == 0)
+  apply_bcs_outerradiation_and_inner(commondata, params, bcstruct, griddata[grid].xx,
+                                     gridfunctions_wavespeed,gridfunctions_f_infinity,
+                                     RK_INPUT_GFS, RK_OUTPUT_GFS);"""
+if not enable_rfm_precompute:
+    rhs_string = rhs_string.replace("rfmstruct", "xx")
+
+MoL.register_CFunctions(
+    MoL_method=MoL_method,
+    rhs_string=rhs_string,
+    post_rhs_string="""if (strncmp(commondata->outer_bc_type, "extrapolation", 50) == 0)
+  apply_bcs_outerextrap_and_inner(commondata, params, bcstruct, RK_OUTPUT_GFS);
+  enforce_detgammabar_equals_detgammahat(commondata, params, rfmstruct, RK_OUTPUT_GFS);""",
+    enable_rfm_precompute=enable_rfm_precompute,
+    enable_curviBCs=True,
+    fp_type=fp_type,
+)
+xxCartxx.register_CFunction__Cart_to_xx_and_nearest_i0i1i2(CoordSystem, fp_type=fp_type)
+xxCartxx.register_CFunction_xx_to_Cart(CoordSystem, fp_type=fp_type)
+progress.register_CFunction_progress_indicator()
+rfm_wrapper_functions.register_CFunctions_CoordSystem_wrapper_funcs()
+
+#########################################################
+# STEP 3: Generate header files, register C functions and
+#         command line parameters, set up boundary conditions,
+#         and create a Makefile for this project.
+#         Project is output to project/[project_name]/
+if CoordSystem == "SinhSpherical":
+    par.adjust_CodeParam_default("SINHW", 0.4)
+par.adjust_CodeParam_default("eta", GammaDriving_eta)
+
+CPs.write_CodeParameters_h_files(project_dir=project_dir)
+CPs.register_CFunctions_params_commondata_struct_set_to_default()
+cmdpar.generate_default_parfile(project_dir=project_dir, project_name=project_name)
+cmdpar.register_CFunction_cmdline_input_and_parfile_parser(
+    project_name=project_name, cmdline_inputs=["convergence_factor"]
+)
+Bdefines_h.output_BHaH_defines_h(
+    project_dir=project_dir,
+    enable_simd=enable_simd,
+    enable_rfm_precompute=enable_rfm_precompute,
+    fin_NGHOSTS_add_one_for_upwinding_or_KO=True,
+    REAL_means=fp_type,
+)
+
+main.register_CFunction_main_c(
+    initial_data_desc=IDtype,
+    MoL_method=MoL_method,
+    boundary_conditions_desc=boundary_conditions_desc,
+    set_initial_data_after_auxevol_malloc=True,
+)
+griddata_commondata.register_CFunction_griddata_free(
+    enable_rfm_precompute=enable_rfm_precompute, enable_CurviBCs=True
+)
+
+if enable_simd:
+    copy_files(
+        package="nrpy.helpers",
+        filenames_list=["simd_intrinsics.h"],
+        project_dir=project_dir,
+        subdirectory="simd",
+    )
+Makefile.output_CFunctions_function_prototypes_and_construct_Makefile(
+    project_dir=project_dir,
+    project_name=project_name,
+    exec_or_library_name=project_name,
+    compiler_opt_option="default",
+)
+shutil.copyfile("tov_id/tov.asc", f"project/{project_name}/tov.asc")
```

Comment:
Data files are not permitted in NRPy 2. Instead we need to port over the TOV solver from NRPy 1, and adjust this example file to generate the data file. Ideally we just use a simple gamma-law EOS.

Reply:
Update: TOVola has been imported into NRPy 2! Check out the example in `nrpy/examples/tovola_neutron_star.py`

Reply:
It currently only supports simple polytropes (to sidestep GRHayL dependence for now), but hey, that's a start.